packages/api-db-mongodb/src/db/invoice.ts

```diff
@@ -232,5 +231,9 @@ function invoiceDateForSort(invoice: DBInvoice, sort: InvoiceSort): Date {
 
     case InvoiceSort.PaidAt:
       if (invoice.paidAt) return invoice.paidAt
```

Comment:
I'd recommend something like `return invoice.paidAt ? invoice.paidAt : invoice.createdAt` here. With this you don't need the `... | undefined` on line 224.

Reply:
Thanks. Just changed it as per your recommendation. I've also changed the `switch` `default` case to `return invoice.createdAt`. Happy to apply more changes if needed.